blaze/expr/collections.py

```diff
@@ -425,9 +432,26 @@ def join(lhs, rhs, on_left=None, on_right=None, how='inner'):
 join.__doc__ = Join.__doc__
 
 
-from .expressions import dshape_method_list
+class IsIn(ElemWise):
+    """Return a boolean expression indicating whether another expression
+    contains values that are members of a collection.
+    """
+    __slots__ = '_hash', '_child', '_keys'
+
+    @property
+    def schema(self):
+        return datashape.bool_
+
+
+def isin(child, keys):
+    return IsIn(child, frozenset(keys))
```

Comment:
I'm slightly concerned about people using an expression for `keys`.  Request informative error message.